### ISSUE TITLE
code

### ISSUE THREAD

**Issue by yican-huang** [chenjzBUAA/SATIR#2]
Great job. After reading the article, I was very impressed and interested. I'm not sure if authors have any plans to open the training code. Looking forward to it!

**Comment by chenjzBUAA** [comment]
Hi,there.
This paper focuses on the strategy of generating pseudo-labels for pretraining using SAM.
As shown in Figures 1 and 2, the pre-training and training models are actually the models that you intend to use. Our pseudo-label generation strategy, combined with the dataset (or your own unlabeld datasets in your applications), can be employed for pre-training purposes.
